src/iota/bundle.c

```diff
@@ -45,11 +50,19 @@ void bundle_set_address_bytes(BUNDLE_CTX *ctx, const unsigned char *addresses)
     os_memcpy(bytes_ptr, addresses, 48);
 }
 
+trit_t bundle_essence_trits[243];
```

Comment:
What are the benefits of having a global variable instead of a stack variable, here and in the other places?
The stack variable only consumes memory, when in that function. Is the stack more limited?

Reply:
I had a discussion in a [RIOT issue. ](https://github.com/RIOT-OS/RIOT/issues/10081#issuecomment-425678888)
I like the idea of using a global variable and mutex to make it safe for multi threading. 
She arguments especially with the [inability of growing and shrinking the memory usage.](https://github.com/RIOT-OS/RIOT/issues/10081#issuecomment-425679455) Since a lot of MCUs don't have a MMU that sounds reasonable for me.